stratum/hal/lib/common/yang_parse_tree_paths.cc

```diff
@@ -881,6 +881,27 @@ void SetUpInterfacesInterfaceStateAdminStatus(uint64 node_id, uint32 port_id,
       ->SetOnChangeHandler(on_change_functor);
 }
 
+////////////////////////////////////////////////////////////////////////////////
+// /interfaces/interface[name=<name>]/state/loopback-mode
+//
+void SetUpInterfacesInterfaceStateLoopbackStatus(uint64 node_id, uint32 port_id,
```

Comment:
I would suggest using `SetUpInterfacesInterfaceStateLoopbackMode` since the method name is based on the gNMI path